apollo-router/src/cache/storage.rs

```diff
@@ -160,7 +160,7 @@ where
     pub(crate) async fn insert(&self, key: K, value: V) {
         if let Some(redis) = self.redis.as_ref() {
             redis
-                .insert(RedisKey(key.clone()), RedisValue(value.clone()))
+                .insert(RedisKey(key.clone()), RedisValue(value.clone()), None)
```

Comment:
`RedisCacheStorage::insert` has an optional ttl argument because in the entity cache, we may set  different ttl values depending on what the subgraph returned in its `Cache-Control` header.

But `RedisCacheStorage::insert` is also used in `CacheStorage::insert`, the dual in memory/redis cache used in APQ and query plan caches, for which we will use a global TTL instead (cf https://github.com/apollographql/router/pull/4164)